catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1053,6 +1225,37 @@ public QueryResponse query(QueryRequest queryRequest, FederationStrategy strateg
                         queryRequest.getProperties());
             }
 
+            if (policy != null) {
+                HashMap<String, Set<String>> requestPolicyMap = new HashMap<>();
+                Map<String, Serializable> unmodifiableProperties = Collections.unmodifiableMap(
+                        queryReq.getProperties());
+                for (PolicyPlugin plugin : policy) {
+                    try {
+                        PolicyResponse policyResponse = plugin.processPreQuery(queryReq.getQuery(), unmodifiableProperties);
+                        buildPolicyMap(requestPolicyMap, policyResponse.operationPolicy().entrySet());
+                    } catch (PluginExecutionException e) {
+                        LOGGER.info(
+                                "Plugin processing failed. This is allowable. Skipping to next plugin.",
+                                e);
+                    } catch (StopProcessingException e) {
+                        throw new FederationException("Query could not be executed.", e);
+                    }
+                }
+                queryReq.getProperties().put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
+            }
+
+            if (access != null) {
+                for (AccessPlugin plugin : access) {
+                    try {
+                        plugin.processPreQuery(queryReq);
+                    } catch (PluginExecutionException see) {
+                        LOGGER.warn("Error executing CatalogAccessPlugin: " + see.getMessage(), see);
```

Comment:
This is logged as an `info` above and ignored but here it is logged as a warning and the message says it's an error. We should be consistent.